modules/rhel-preparing-playbook-machine.adoc

```diff
@@ -60,10 +60,10 @@ If you use SSH key-based authentication, you must manage the key with an SSH age
 [source,terminal]
 ----
 # subscription-manager repos \
-    --enable="rhel-7-server-rpms" \
-    --enable="rhel-7-server-extras-rpms" \
-    --enable="rhel-7-server-ansible-2.9-rpms" \
-    --enable="rhel-7-server-ose-4.9-rpms"
+    --enable="rhel-8-server-rpms" \
```

Comment:
I had a search in Errata release tool(https://errata.devel.redhat.com/package/show/openshift-ansible) , there's no openshift-ansible el8 package built for RHEL-8, so we may still have to run the playbook on RHEL-7 machines.

The following repos should work:
```
# subscription-manager repos \
    --enable="rhel-7-server-rpms" \
    --enable="rhel-7-server-extras-rpms" \
    --enable="rhel-7-server-ansible-2.9-rpms" \
    --enable="rhel-7-server-ose-4.10-rpms"

```